scripts/install_odb_2.5.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+trap cleanup EXIT
+
+declare ODB_INSTALL_DIR="/usr/local"
+declare ODB_BUILD_DIR="/tmp/odb"
+
+function cleanup() {
+    echo "Cleaning up Odb temporaries."
+    if [[ -n "${ODB_BUILD_DIR}" ]]; then
+        rm --recursive --force "${ODB_BUILD_DIR}"
+    fi
+}
+
+function usage() {
+    cat <<EOF
+This is a convenience script that installs the build2 build system and
+uses that to build and install the odb compiler, runtime library and also
+the slite and pgsql support library.
+
+${0} [-h] [-d] <odb_install_path> 
+  -h  Print this usage information, and quit.
+  -d  Install directory of odb. Optional. /opt/odb is the deafault.
+EOF
+}
+
+while getopts "hd:" OPTION; do
+    case ${OPTION} in
+        h)
+            usage
+            exit 0
+            ;;
+        d)
+            ODB_INSTALL_DIR="${OPTARG}"
+            ;;
+        *)
+            usage >&2
+            exit 1
+            ;;
+    esac
+done
+
+
+#Build2 toolchain
+mkdir /tmp/build2src
+pushd "/tmp/build2src"
+curl -sSfO https://download.build2.org/0.12.0/build2-install-0.12.0.sh
+sh build2-install-0.12.0.sh --yes --trust yes
+popd
+
+
+#Building the odb compiler
+mkdir /tmp/odb
+pushd "/tmp/odb"
+bpkg create --quiet --jobs $(nproc) cc \
+  config.cxx=g++                       \
+  config.cc.coptions=-O3               \
+  config.bin.rpath=/usr/local/lib      \
+  config.install.root=/usr/local       \
+  config.install.root=/usr/local
+
+bpkg add https://pkg.cppget.org/1/beta --trust-yes
+bpkg fetch --trust-yes
+bpkg build odb --yes
+bpkg build libodb --yes
+bpkg build libodb-sqlite --yes
+bpkg build libodb-pgsql --yes
```

Comment:
Is the order of these commands okay? Usually ODB was built with `libodb` first, then `libodb-<database>` second (for which build2 told me to add an extra `config.import.libodb` line!), then the ODB binary itself.

I think this script needs to be tested in cleaned environments.

Reply:
The commands I executed on my machine was:

`bpkg create [... your arguments in the script ...] config.import.libodb=libodb-2.5.0-b.17`
(the directory name mined from the downloaded thingy)
then

 * `bpkg build libodb`
 * `bpkg build libodb-sqlite` (this broke first and required me to add the import line)
 * and finally `bpkg build odb`

then the `install` one.

Reply:
Well I believe the order is okay, but I'm not 100% sure. At the bpkg's web page the tutorial followed this order, that's why I did too. 
About the testing, I completely agree, the script is currently only tested on the ubuntu :bionic docker image.

Reply:
I have this error message on `fetch` command:
```
error: unable to fetch https://pkg.cppget.org/1/beta/repositories.manifest
```

Reply:
Is it possible that you have some network issues? `curl` can download this file successfully: `curl https://pkg.cppget.org/1/beta/repositories.manifest`.